src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/dialogs/rowconfig.html

```diff
@@ -8,7 +8,12 @@
       <div class="alert alert-warn ng-scope" ng-show="nameChanged">
           Modifying a row configuration name will result in loss of
           data for any existing content that is based on this configuration.
+          Modifying only the label will not result in data loss.
```

Comment:
The whole warning is marked in a big yellow warning blob thingy ... and I figured the stuff I added is actually not the "critical" part of that message.. So it didn't feel right to mark my part as strong.. and marking the other part as strong was kinda irrelevant as it already was part of the big yellow thing 😄